apps/charterafrica/src/payload/blocks/Mooc.js

```diff
@@ -0,0 +1,56 @@
+import linkGroup from "../fields/linkGroup";
+import { MuiButtonColors } from "../utils/colors";
+
+const Mooc = {
+  slug: "mooc",
+  fields: [
+    {
+      name: "title",
+      type: "richText",
+      required: true,
+      localized: true,
+      admin: {
+        elements: ["h1"],
+        leaves: ["bold", "italic", "underline", "code"],
+      },
+    },
+    {
+      name: "link",
+      type: "group",
+      fields: [
+        {
+          name: "color",
+          type: "select",
+          options: MuiButtonColors,
+          required: true,
+          label: {
+            en: "Button Color",
+            fr: "Couleur du bouton",
+            pt: "Cor do botão",
+          },
+          admin: {
+            isClearable: true,
+            isSortable: true,
+          },
+        },
```

Comment:
For a millionth time, where does this color ever change? Why do we need it in the CMS?